OptiFineDoc/assets/minecraft/optifine/lang/cs_cz.lang

```diff
@@ -679,11 +681,12 @@ of.options.FULLSCREEN_MODE.tooltip.5=Nižší rozlišení by měli být většin
 
 of.options.SHOW_FPS=Zobrazit FPS
 of.options.SHOW_FPS.tooltip.1=Kompaktně zobrazí informace o FPS a renderu.
-of.options.SHOW_FPS.tooltip.2=  C: - načítání chunků
-of.options.SHOW_FPS.tooltip.3=  E: - načtené entity + blokové entity
-of.options.SHOW_FPS.tooltip.4=  U: - přednačtené chunky
-of.options.SHOW_FPS.tooltip.5=Informace o FPS se zobrazí jen,
-of.options.SHOW_FPS.tooltip.6=když je ladící obrazovka (F3) zneviditelněná.
+of.options.SHOW_FPS.tooltip.2=  Fps - průměr/minimum

```

Comment:
Shouldn't `FPS` be in uppercase?

Reply:
It should be, but it's not like that in en_us.lang.
But I'm going to make it uppercase anyway, I think it looks better.